gammapy/scripts/tests/test_image_pipe.py

```diff
@@ -57,7 +58,8 @@ def test_image_pipe(tmpdir):
     refheader = image.to_image_hdu().header
     exclusion_mask = ExclusionMask.read('$GAMMAPY_EXTRA/datasets/exclusion_masks/tevcat_exclusion.fits')
     exclusion_mask = exclusion_mask.reproject(reference=refheader)
-    images = ImageAnalysis(image, energy_band=energy_band, offset_band=offset_band, data_store=data_store,
-                           obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
-
-    images.make_maps(radius=10., bkg_norm=True, spectral_index=2.3, for_integral_flux=True)
+    # Pb with the load psftable for one of the run that is not implemented yet...
+    data_store.hdu_table.remove_row(14)
+    mosaic_images = MosaicImage(image, energy_band=energy_band, offset_band=offset_band, data_store=data_store,
+                                obs_table=data_store.obs_table, exclusion_mask=exclusion_mask)
+    mosaic_images.make_images(make_background_image=True, for_integral_flux=True, radius=10.)
```

Comment:
Can you add at least some assert statements here, testing on the sum of the data? E.g.:

```
from numpy.testing.utils import assert_equal
assert_equal(mosaic.maps['counts'].data.sum(), 42)
assert_equal(mosaic.maps['background'].data.sum(), 84.123456)
assert_equal(mosaic.maps['significance'].lookup(crab_position), 50.3455)
...
```
This will pin the status of the code to the version you are using right now and make your (science) results reproducible.